typeDefinitions/answers.d.ts

```diff
@@ -0,0 +1,41 @@
+import { Response } from "express";
+import * as admin from "firebase-admin";
+
+export type AnswerBody = {
+  id: string;
+  eventId: string;
+  answeredBy: string;
+  answer: string;
+  questionId: string;
+};
+
+export type AnswerStatus = "PENDING" | "APPROVED" | "REJECTED";
+
+export type AnswerFieldsToUpdate = {
+  status?: AnswerStatus;
```

Comment:
Question: Why is status optional? What else is updated after review other than status?

Reply:
kept is optional because it's a patch request in future may be we want to update more properties and we are free to skip this as optional or others optional(in case of updating status)